CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Next
 * Added Travis-CI - [@dblock](https://github.com/dblock).
 * Added TODO list and enhanced README documentation - [@ptitvert](https://github.com/ptitvert)
 * [#15](https://github.com/dblock/oshi/pull/15): Added support for CPU load on OSX and Linux - [@kamenitxan](https://github.com/kamenitxan).
+* Added support for CPU load on Windows [@sorceror](https://github.com/Sorceror).
```

Comment:
Amend the line above to be something like this:

```
* [#15](https://github.com/dblock/oshi/pull/15), [#16](https://github.com/dblock/oshi/pull/16): Added support for CPU load - [@kamenitxan](https://github.com/kamenitxan), [@Sorceror](https://github.com/Sorceror).
```

Users just care that we fully support CPU load ;)